src/erlcloud_sns.erl

```diff
@@ -192,28 +194,28 @@ confirm_subscription2(Token, TopicArn, Config) ->
     erlcloud_xml:get_text(
         "ConfirmSubscriptionResult/SubscriptionArn", Doc).
 
--spec confirm_subscription2/4 :: (string(), string(), string(), string()) -> string().
+-spec confirm_subscription2/4 :: (parameter(), parameter(), parameter(), parameter()) -> string().
 confirm_subscription2(Token, TopicArn, AccessKeyID, SecretAccessKey) ->
     confirm_subscription2(Token, TopicArn, new_config(AccessKeyID, SecretAccessKey)).
 
 
 
--spec delete_endpoint/1 :: (string()) -> ok.
+-spec delete_endpoint/1 :: (parameter()) -> ok.
 delete_endpoint(EndpointArn) ->
     delete_endpoint(EndpointArn, default_config()).
 
--spec delete_endpoint/2 :: (string(), aws_config()) -> ok.
+-spec delete_endpoint/2 :: (parameter(), aws_config()) -> ok.
 delete_endpoint(EndpointArn, Config) ->
     sns_simple_request(Config, "DeleteEndpoint", [{"EndpointArn", EndpointArn}]).
```

Comment:
Line 209 is too long:     sns_simple_request(Config, "DeleteEndpoint", [{"EndpointArn", EndpointArn}])..